_includes/integrations/react-tech-ref.md

```diff
@@ -0,0 +1,361 @@
+  - [`id`](#id)
+  - [`init`](#init)
+  - [`initialValue`](#initialvalue)
+  - [`inline`](#inline)
+  - [`onEditorChange`](#oneditorchange)
+  - [`plugins`](#plugins)
+  - [`tagName`](#tagname)
+  - [`textareaName`](#textareaname)
+  - [`toolbar`](#toolbar)
+  - [`value`](#value)
+
+- [Using the TinyMCE React component as a controlled component](#usingthetinymcereactcomponentasacontrolledcomponent)
+- [Event binding](#eventbinding)
+
+### Installing the TinyMCE React integration using NPM
+
+To install the `tinymce-react` package and save it to your `package.json`.
+
+    ```
+    $ npm install --save @tinymce/tinymce-react
+    ```
+
+### Configuring the editor
+
+The editor accepts the following properties:
+
+```xml
+<Editor
+  apiKey='your-api-key'
+  cloudChannel='5-stable'
+  disabled={false}
+  id='uuid'
+  init= {% raw %}{{{% endraw %}  {% raw %}}}{% endraw %}
+  initialValue=''
+  inline={false}
+  onEditorChange={}
+  plugins=''
+  tagName='div'
+  textareaName=''
+  toolbar=''
+  value=''
+/>
+```
+
+None of the configuration properties are **required** for `tinymce-react` to work. Specify a Tiny Cloud API key using `apiKey` to remove the `This domain is not registered...` warning message.
+
+#### `apiKey`
+Tiny API key. Required for deployments using the Tiny Cloud to provide the TinyMCE editor.
+
+To register for a Tiny Cloud API key, visit the [sign-up page](https://www.tiny.cloud/signup/).
+
+**Default value:** `no-api-key`
+
+**Type:** String
+
+##### Example: `apiKey`
+
+```xml
+<Editor
+  apiKey='your-api-key'
+/>
+```
+
+#### `cloudChannel`
+
+**Default value:** `5-stable`
+
+**Possible values:**  `5-stable`, `5-testing`, `5-dev`
+
+Changes the TinyMCE build used for the editor to one of the following cloud channels:
+
+- `5-stable` (**Default**): The current enterprise release of TinyMCE.
+- `5-testing`: The current release candidate for the next enterprise release of TinyMCE.
+- `5-dev`: The nightly-build version of TinyMCE.
+
+Such as:
+
+```js
+<Editor apiKey='your-api-key' cloudChannel='5-dev' init={% raw %}{{{% endraw %} /* your other settings */ {% raw %}}}{% endraw %} />
+```
+For information TinyMCE development channels, see: [Specifying the TinyMCE editor version deployed from Cloud - dev, testing, and stable releases]({{site.baseurl}}/cloud-deployment-guide/editor-plugin-version/#devtestingandstablereleases).
+
+#### `disabled`
+The `disabled` property can dynamically switch the editor between a "disabled" (read-only) mode (`true`) and the standard editable mode (`false`).
+
+**Default value:** `false`
+
+**Possible values:**  `true`, `false`
+
+##### Example: `disabled`
+
+```xml
+<Editor
+  disabled={true}
+/>
+```
+
+#### `id`
+An id for the editor. Used for retrieving the editor instance using the `tinymce.get('ID')` method. Defaults to an automatically generated [UUID](https://tools.ietf.org/html/rfc4122).
+
+**Default value:** Automatically generated [UUID](https://tools.ietf.org/html/rfc4122).
```

Comment:
in the example, we have a string 'uuid' as the placeholder, is that the default value for automatic UUID?